packages/moti/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "extends": "expo-module-scripts/tsconfig.base",
   "compilerOptions": {
-    "jsx": "react-jsx",
```

Comment:
The `expo-module-scripts/tsconfig.base.json` file already sets `jsx: react-native`.